lib/mobius/validations/event_validator.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Mobius.Validations.EventValidator do
+  @moduledoc false
+
+  alias Mobius.Core.Event
+  alias Mobius.Validations.Utils
+
+  @spec validate_names([any]) :: Utils.out()
```

Comment:
```suggestion
  @spec validate_names([Event.names()]) :: Utils.out()
```

We're expecting these to be event names aren't we?

Reply:
We're expecting them to be, but it can be anything. If it's an event name, it spits out an `:ok`, otherwise it spits out an `{:errors, [String.t()]}` tuple. So the input here is any even if a "good" case is for `Event.names()` to be given

Reply:
I guess this means `Mobius.Actions.Events` should probably also take `any` 🤔